website/content/docs/agent-and-proxy/proxy/caching/static-secret-caching.mdx

```diff
@@ -0,0 +1,140 @@
+---
+layout: docs
```

Comment:
That makes sense, thanks! It's intended to be a technical overview, but I included information on e.g. the required auto-auth permissions because enabling event system access is quite novel. We don't have any other subsystem of Vault that requires it today, as far as I know? I'm incorporating your feedback, I appreciate it all! :D